pkg/cmd/openshift/openshift.go

```diff
@@ -52,6 +53,8 @@ func CommandFor(basename string) *cobra.Command {
 		cmd = irouter.NewCommandTemplateRouter(basename)
 	case "openshift-deploy":
 		cmd = deployer.NewCommandDeployer(basename)
+	case "openshift-diagnostics":
```

Comment:
let's keep it as an experimental subcommand for now... top-level commands require install changes to actually be available. was there agreement that we wanted a top-level command for this one?